src/main/webapp/scheduled-interviews.js

```diff
@@ -12,6 +12,95 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
 function onScheduledInterviewsLoad() {
   supplyLogoutLink();
+  listScheduledInterviews(); 
+}
+
+function listScheduledInterviews() {
+  var email = "gswe@gmail.com"; 
+  const scheduledInterviewsSection = document.getElementById('scheduled-interviews-cards');
+  document.getElementById('scheduled-interviews-cards').innerHTML = ''; 
+  fetch(`/scheduled-interviews?userEmail=${email}`)
+    .then(response => response.json())
+    .then((scheduledInterviews) => {
+      if (isEmptyObject(scheduledInterviews)){
+        const emptyEventsMessageDiv = document.createElement('div');
+        const  emptyEventsMessage = document.createElement('h2');
+        emptyEventsMessage.style.textAlign = 'center';  
+        emptyEventsMessage.innerHTML = "No Scheduled Interviews"; 
+        emptyEventsMessageDiv.appendChild(emptyEventsMessage); 
+        scheduledInterviewsSection.appendChild(emptyEventsMessage); 
```

Comment:
Would this be better as a hidden div that can then be revealed? If the text is always the same you can create in html instead.